src/openshift/component.ts

```diff
@@ -163,6 +163,15 @@ export class Component extends OpenShiftItem {
         return null;
     }
 
+    private static async validateComponentName(value: string, application: OpenShiftObject) {
```

Comment:
I am sure you have one project, one application and one component. This is the only condition when it works.